extensions/wikia/PhalanxII/classes/PhalanxStatsPager.class.php

```diff
@@ -45,17 +45,28 @@ function getPagingQueries() {
 	}
 
 	function formatRow( $row ) {
-		$type = implode( ", ", Phalanx::getTypeNames( $row->ps_blocker_hit ?? $row->ps_blocker_type ) );
+		$blocker = $row->ps_blocker_hit ?: $row->ps_blocker_type;
+		$type = implode( ', ', Phalanx::getTypeNames( $blocker ) );
```

Comment:
Use the elvis operator rather than the null coalescing operator as `ps_blocker_hit` can sometimes be zero which is just as invalid as `null` in this case.